database/src/migrations/20210818170006_add_geojson_to_project_and_survey.ts

```diff
@@ -0,0 +1,39 @@
+import * as Knex from 'knex';
+
+const DB_SCHEMA = process.env.DB_SCHEMA;
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`
+    set schema '${DB_SCHEMA}';
+    set search_path = ${DB_SCHEMA},public;
+
+    ALTER TABLE ${DB_SCHEMA}.project add column geojson jsonb;
+    ALTER TABLE ${DB_SCHEMA}.survey add column geojson jsonb;
+
+    set search_path = biohub_dapi_v1;
+
+    set role biohub_api;
+
+    create or replace view project as select * from ${DB_SCHEMA}.project;
+    create or replace view survey as select * from ${DB_SCHEMA}.survey;
+
+    set role postgres;
+  `);
+}
+
+export async function down(knex: Knex): Promise<void> {
```

Comment:
technically this isnt quite right, as the views cant jsut be dropped, they need to be updated (just the reverse of what you have in the up):

1. alter table drop geojson column
2. update views

Reply:
spoke to charlie - he said not to worry about the down at all
but I can make the update

Reply:
no worries, either way, just wanted to point it out in case